packages/admin/cms-admin/src/redirects/createRedirectsPage.tsx

```diff
@@ -12,17 +12,21 @@ import { RedirectForm } from "./RedirectForm";
 import { RedirectsGrid } from "./RedirectsGrid";
 
 interface CreateRedirectsPageOptions {
-    customTargets?: Record<string, BlockInterface>;
+    customTargets?: Record<string, BlockInterface>; // TODO: Remove customTargets here and instead use createRedirectLinkBlock to create a custom link block for the redirects
     scopeParts?: string[];
 }
 
-function createRedirectsPage({ customTargets, scopeParts = [] }: CreateRedirectsPageOptions = {}): ComponentType {
-    const linkBlock = createOneOfBlock({
+export function createRedirectLinkBlock(customTargets?: Record<string, BlockInterface>) {
```

Comment:
Please rename it to createRedirect**s**LinkBlock

Reply:
changed in: 318b871de3979338c99df8122f480ce030705039